eng/Directory.Build.Common.props

```diff
@@ -243,6 +243,14 @@
     <RunSettingsFilePath Condition="'$(AZURE_SKIP_DEFAULT_RUN_SETTINGS)' != 'true'">$(RepoEngPath)\nunit.runsettings</RunSettingsFilePath>
   </PropertyGroup>
 
+  <!-- Set global test timeout of 5 seconds -->
+  <ItemGroup Condition="'$(IsTestProject)' == 'true'">
+    <AssemblyAttribute Include="NUnit.Framework.Timeout">
```

Comment:
Do we have test projects that are not using NUnit? If there are then this will likely fail to compile for them. I'm thinking about track 1 libraries for example.

Reply:
Yes, I made a change just now to eliminate track1 with the IsClientLibrary condition.

Reply:
I would use the IsClientLIbrary variable directly instead of trying to duplicate the condition hear.

Reply:
I did initially, but I think some track1 clients set this property directly and the name check eliminates those.

Reply:
If there are libraries setting that directly then they should not be track 1 and so they should be considered track 2 and we should probably treat them the same way.

Reply:
Did you take into account that we have many track 2s that use "Microsoft.Azure." as a prefix, though? It would be hard to eliminate them all. E.g. sdk/core/Microsoft.Azure.Core.Spatial.

Reply:
Agree. The evaluation condition of $(IsClientLibrary) is unfortunately not entirely correct. So, we are forced to set it to true in some track 2 libraries.
So, the current best option is to continue using this one variable, so we don't accidentally exclude any track 2 libraries.

Reply:
I've implemented it as opt out now using the new `IsOptOutOfGlobalTestTimeout` flag.